clang/include/clang/CConv/ConstraintBuilder.h

```diff
@@ -23,6 +67,8 @@ class ConstraintBuilderConsumer : public clang::ASTConsumer {
 
 private:
   ProgramInfo &Info;
+  void SetProgramInfoTypeVars(TypeVariableMapT TypeVariableBindings,
+                              ASTContext &C);
 };
 
 #endif
```

Comment:
This is looking like a good candidate to put in its own file, along with other stuff relating to generics and type variables?